util/topsql/topsql.go

```diff
@@ -50,17 +50,19 @@ func SetupTopSQL() {
 	singleTargetDataSink = reporter.NewSingleTargetDataSink(remoteReporter)
 
 	globalTopSQLReport = remoteReporter
-
-	tracecpu.GlobalSQLCPUProfiler.SetCollector(remoteReporter)
-	tracecpu.GlobalSQLCPUProfiler.Run()
 	stmtstats.RegisterCollector(remoteReporter)
 	stmtstats.SetupAggregator()
 }
```

Comment:
How about use a init() to assign the `globalTopSQLReport`? Then you don't need to worry about it is possibly nil. You can start its internal goroutine later in `SetupTopSQL`.

Reply:
done.